lib/tasks/button_ids.rake

```diff
@@ -0,0 +1,10 @@
+namespace :champaign do
+  desc "Associate existing shares with their respective button_ids"
+  task associate_buttons: :environment do
+    Share::Variant.all.each do |variant|
+      # ALERT - this needs good testing before running it on production
```

Comment:
I ran it a few times locally after finding out that testing rake tasks is a nightmare. This could be done as a one-off command, without the task - and the button_ids on both staging and production at the moment are all `nil` so it'd be pretty easy to revert.